plugins/woocommerce/src/Admin/API/Reports/GenericController.php

```diff
@@ -22,6 +22,12 @@ abstract class GenericController extends \WC_REST_Reports_Controller {
 	 */
 	protected $namespace = 'wc-analytics';
 
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'reports';
```

Comment:
If we could disregard any backward compatibility, IMHO, it would be better to use `''` here to distinguish from the endpoint to list all endpoints.